api/help/help-online-editor_en.php

```diff
@@ -0,0 +1,7 @@
+<iframe src="https://wiki.openseamap.org/wiki/Online-Editor" width="540" height="670" frameborder="0">
+<p>Your browser can not show embedded frames:
+You may view the embedded site over the following
```

Comment:
Minor: I think "through the following" would be a better wording. And you can use "link" instead of "reference".

Reply:
Same comment for all the following _en pages...

Reply:
fixed